CMakeLists.txt

```diff
@@ -104,10 +114,21 @@ if(BUILD_TESTS)
 endif()
 
 set(SYSTEMD_UNITDIR lib/systemd/system CACHE PATH "Path to systemd unit files")
+
 set(BASH_COMPLETIONS_DIR usr/share/bash-completion/completions CACHE PATH "Path to bash completions directory")
 set(ZSH_COMPLETIONS_DIR usr/share/zsh/site-functions CACHE PATH "Path to zsh completions directory")
-install(TARGETS laminard laminarc RUNTIME DESTINATION usr/bin)
-install(FILES etc/laminar.service DESTINATION ${SYSTEMD_UNITDIR})
-install(FILES etc/laminar.conf DESTINATION etc)
 install(FILES etc/laminarc-completion.bash DESTINATION ${BASH_COMPLETIONS_DIR} RENAME laminarc)
 install(FILES etc/laminarc-completion.zsh DESTINATION ${ZSH_COMPLETIONS_DIR} RENAME _laminarc)
+
+install(TARGETS laminard laminarc RUNTIME DESTINATION usr/bin)
+install(FILES etc/laminar.service DESTINATION ${SYSTEMD_UNITDIR})
+
+# Don't overwrite the config file if it already exists at the target location
+# @todo - migration/override script if the defaults change or the options get pruned?
+if(NOT EXISTS "${CMAKE_INSTALL_PREFIX}etc/laminar.conf")
+    install(
+        FILES etc/laminar.conf
+        DESTINATION ${CMAKE_INSTALL_PREFIX}etc
+    )
+endif(NOT EXISTS "${CMAKE_INSTALL_PREFIX}etc/laminar.conf")
```

Comment:
Again, you can't really do this. CMake runs at configuration time, and if you are creating a package to install on a different machine (which CMake cannot detect), the file should always be installed, even if the host computer has one already.

Reply:
That's why I changed it to check at the `CMAKE_INSTALL_PREFIX` location. Even if the system the user is compiling on already has an `/etc/laminar.conf`, this should correctly output if the prefix tells `make` to output to somewhere other than `/etc`.

I could also be wrong, and it's _not_ possible to do both local installs and packaging cleanly, but it's late and I only hacked on this enough to un-block my progress on something else.

Barring a fire that takes my attention elsewhere, I'll poke it again tomorrow and see what I can come up with.

Reply:
`CMAKE_INSTALL_PREFIX` != `DESTDIR`

`make install` installs to `$ENV{DESTDIR}/$CMAKE_INSTALL_PREFIX/$x` where `$x` is the path given to cmake's `install`